platforms-serverless-vercel/vercel-cli-cache-repro/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "e2e-vercel-api-cache-repro",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "prisma": "4.10.0-integration-remove-install-now-hack.1",
+    "ts-node": "10.9.1",
+    "vercel": "28.13.1"
+  },
+  "dependencies": {
+    "@prisma/client": "4.10.0-integration-remove-install-now-hack.1",
```

Comment:
This would be changed automatically after merge by the update script.
Is this relevant for the test result?

Reply:
Only the logs are slightly different, but everything should be the same